### PR TITLE
add api to select installation disk

### PR DESCRIPTION
**Pull request by slaviered**

### CODE REVIEW

File: discovery-infra/test_infra/assisted_service_api.py [hunk 1]
```diff
@@ -157,6 +157,20 @@ def update_hosts(self, cluster_id, hosts_with_roles, hosts_names=None):
             cluster_id=cluster_id, cluster_update_params=hosts
         )
 
+    def select_installation_disk(self, cluster_id, hosts_with_diskpaths):
```
Comment (nshidlin) [inline discovery-infra/test_infra/assisted_service_api.py:160]: Please add corresponding  method to `cluster`

**Review by YuviGold — APPROVED** [review]

**Pull request merged**